test/tests/deformation/mesh-velocity-areas/pipe.i

```diff
@@ -0,0 +1,200 @@
+[GlobalParams]
+  use_displaced_mesh = true
+  displacements = 'disp_x_o disp_y_o disp_z_o'
+[]
+
+[Mesh]
+  type = FileMesh
+  file = pipe.exo
+  parallel_type = replicated
+[]
+
+[Variables]
+  [temp]
+    initial_condition = 0.0
+    order = FIRST
+  []
+[]
+
+[Kernels]
+  [heat]
+    type = HeatConduction
+    variable = temp
+  []
+  [heat_source]
+    type = CoupledForce
+    variable = temp
+    v = 0.0
```

Comment:
Why not delete this kernel altogether? If the heat source is zero, the residual is zero?